src/main/c/src/SerialImp.c

```diff
@@ -4375,7 +4375,7 @@ JNIEXPORT jboolean  JNICALL RXTXCommDriver(testRead)(
 	int pid = -1;
 #endif
 #ifdef TRENT_IS_HERE_DEBUGGING_ENUMERATION
-	char message[80];
+	char message[128];
```

Comment:
The important thing in this whole change is truncating rather than overflowing when the message length would exceed the buffer size. That being said, I don't think anyone would begrudge you a few more bytes if you wanted to bump up these buffers up to 256 bytes. Stack space is cheap these days.